bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/AbstractHomematicGateway.java

```diff
@@ -937,4 +937,4 @@ private void handleInvalidConnection(String cause) throws IOException {
             startServers();
         }
     }
-}
+}
```

Comment:
The newline at the end of the file is gone?

Reply:
Oups should be fixed now.